server/middleware/authorisationMiddleware.ts

```diff
@@ -6,7 +6,7 @@ import logger from '../../log'
 export default function authorisationMiddleware(authorisedRoles: string[] = []): RequestHandler {
   return (req, res, next) => {
     if (res.locals && res.locals.user && res.locals.user.token) {
-      const { authorities: roles = [] } = jwtDecode(res.locals.user.token)
+      const { authorities: roles = [] } = jwtDecode(res.locals.user.token) as { authorities?: string[] }
```

Comment:
`jwtDecode` returns unknown - overriding this to say that whatever it returns may or may not contain an array of strings